src/sentry/static/sentry/app/components/events/eventTags/eventTagsPill.tsx

```diff
@@ -73,6 +79,22 @@ const EventTagsPill = ({
           </VersionHoverCard>
         </div>
       )}
+      {isTrace && hasQueryFeature && (
```

Comment:
This will work but if in the future you want to avoid prop drilling for feature flag checks you can use the `<Feature>` component to check for a feature, and render a replacement element or nothing.